server/client-build/service-worker.js

```diff
@@ -14,13 +14,13 @@
 importScripts("https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js");
```

Comment:
As mentioned in another review, we should get not check in the entire `client-build` to git, as it's purely a build artifact. Not necessarily a requirement for this PR, but I think this should be simple to address and minimizes the amount of files and code to wade through in future PR reviews :-)

Reply:
I added this file to the gitignore for now. I can remove these files in different PR